comp/metadata/inventoryagent/inventoryagent.go

```diff
@@ -241,3 +241,12 @@ func (ia *inventoryagent) getPayload() marshaler.JSONMarshaler {
 		Metadata:  data,
 	}
 }
+
+// Get returns a copy of the agent metadata. Useful to be incorporated in the status page.
+func (ia *inventoryagent) Get() map[string]interface{} {
+	data := map[string]interface{}{}
+	for k, v := range ia.data {
+		data[k] = v
```

Comment:
Is it OK if this function shallow copies `ia.data` because `v` can be a `map[string]interface{}` which is a reference?

Reply:
Yes, we explicitly call this out in the `Set` method docstring.